docs/glossary.md

```diff
@@ -30,25 +30,36 @@ Here are some definitions of some key ideas encountered in this documentation.
 tree
 : A "gene tree", i.e., the genealogical tree describing how a collection of
   genomes (usually at the tips of the tree) are related to each other at some
-  chromosomal location. See {ref}`sec_nodes_or_individuals` for discussion
-  of what a "genome" is.
+  chromosomal {ref}`position <sec_data_model_definitions_position>` or location.
+  As the trees may vary depending on this location, they are also known as "local
+  trees". See {ref}`sec_nodes_or_individuals` for discussion of what a "genome" is.
 
 (sec_data_model_definitions_tree_sequence)=
 
 tree sequence
-: A "succinct tree sequence" (or tree sequence, for brevity) is an efficient
-  encoding of a sequence of correlated trees, such as one encounters looking
-  at the gene trees along a genome. A tree sequence efficiently captures the
-  structure shared by adjacent trees, (essentially) storing only what differs
-  between them.
+: A "succinct tree sequence" (or tree sequence, for brevity) is an object
+  that stores the genetic ancestry and mutational history of a set of
+  aligned DNA sequences or genomes. The name reflects the idea that a common
+  way to treat genetic ancestry is as a sequence of correlated
+  {ref}`trees <sec_data_model_definitions_tree>` at different chromosomal
+  {ref}`positions <sec_data_model_definitions_position>`.
+  Branches that are shared between these trees are efficiently stored as a
+  single {ref}`edge <sec_data_model_definitions_edge>`, and adjacent trees
+  may differ by only a few such edges. These edges connect
+  {ref}`nodes <sec_data_model_definitions_node>` (genomes) in
+  the tree sequence, forming a structure which is technically known as a
+  network or graph. Graphs of this sort are sometimes called ancestral
```

Comment:
```suggestion
  the tree sequence, forming  a
  network or graph. Graphs of this sort are sometimes called ancestral
```

If someone knows what a tree is they know what a graph is, "technically" is unnecessary here. If you really don't think it's obvious then link to a wikipedia page.